test/T42.stl.names.multi.ref

```diff
@@ -22,34 +22,26 @@
 
 void bind_std_stl_function(std::function< pybind11::module &(std::string const &namespace_) > &M)
 {
-	{ // std::binary_function file:bits/stl_function.h line:114
+	{ // std::binary_function file:bits/stl_function.h line:118
 		pybind11::class_<std::binary_function<float,float,bool>, std::shared_ptr<std::binary_function<float,float,bool>>> cl(M("std"), "binary_function_float_float_bool_t", "");
 		cl.def( pybind11::init( [](std::binary_function<float,float,bool> const &o){ return new std::binary_function<float,float,bool>(o); } ) );
 		cl.def( pybind11::init( [](){ return new std::binary_function<float,float,bool>(); } ) );
-		cl.def("assign", (struct std::binary_function<float, float, bool> & (std::binary_function<float,float,bool>::*)(const struct std::binary_function<float, float, bool> &)) &std::binary_function<float, float, bool>::operator=, "C++: std::binary_function<float, float, bool>::operator=(const struct std::binary_function<float, float, bool> &) --> struct std::binary_function<float, float, bool> &", pybind11::return_value_policy::automatic, pybind11::arg(""));
 	}
-	{ // std::equal_to file:bits/stl_function.h line:204
+	{ // std::equal_to file:bits/stl_function.h line:351
 		pybind11::class_<std::equal_to<float>, std::shared_ptr<std::equal_to<float>>, std::binary_function<float,float,bool>> cl(M("std"), "equal_to_float_t", "");
 		cl.def( pybind11::init( [](){ return new std::equal_to<float>(); } ) );
 		cl.def("__call__", (bool (std::equal_to<float>::*)(const float &, const float &) const) &std::equal_to<float>::operator(), "C++: std::equal_to<float>::operator()(const float &, const float &) const --> bool", pybind11::arg("__x"), pybind11::arg("__y"));
-		cl.def("assign", (struct std::binary_function<float, float, bool> & (std::binary_function<float,float,bool>::*)(const struct std::binary_function<float, float, bool> &)) &std::binary_function<float, float, bool>::operator=, "C++: std::binary_function<float, float, bool>::operator=(const struct std::binary_function<float, float, bool> &) --> struct std::binary_function<float, float, bool> &", pybind11::return_value_policy::automatic, pybind11::arg(""));
 	}
 }
 
 
 // File: std/functional_hash.cpp
-#include <functional> // std::_Manager_operation
-#include <functional> // std::bad_function_call
+#include <bits/std_function.h> // std::_Manager_operation
+#include <bits/std_function.h> // std::bad_function_call
```

Comment:
This looks wrong, could you please tell me what OS/LLVM version have you used to produce this reference results? I want to run this locally because it looks like include-resolution code will need adjustments.

Reply:
That was RHEL8 
gcc -8.3.1
llvm 8.0.1

Reply:
Thank you! I will try to replicate this locally.

Reply:
ok, fix for this is now in master